ReactiveCocoa/Swift/ObjectiveCBridging.swift

```diff
@@ -147,14 +147,14 @@ extension RACCommand {
 
 		enabledProperty <~ self.enabled.toSignalProducer()
 			|> map { $0 as! Bool }
-			|> catch { _ in SignalProducer<Bool, NoError>(value: false) }
+			|> `catch` { _ in SignalProducer<Bool, NoError>(value: false) }
```

Comment:
I prefer @jspahrsummers of `flatMapErrors` for consistency with the other `map` functions